src/components/view/HealthAndWellnessView/HealthAndWellnessView.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react'
+import { Layout, StatusBarBackground, LabResultsSummary, ExternalAccountsPreview, ConnectEhr, Card, Section } from "../.."
+import MyDataHelps from '@careevolution/mydatahelps-js';
+import { TermInformation } from '../../presentational/LabResultWithSparkline/LabResultWithSparkline';
+import HealthPreviewSection from '../../container/HealthPreviewSection/HealthPreviewSection';
+import ExternalAccountsLoadingIndicator from '../../container/ExternalAccountsLoadingIndicator';
+
+export interface HealthAndWellnessViewProps {
+    previewState?: "default"
+    colorScheme?: "auto" | "light" | "dark"
+    connectEhrApplicationUrl: string
+    externalAccountsApplicationUrl: string
+    variant?: "default" | "cardBased"
+}
+
+export default function (props: HealthAndWellnessViewProps) {
+    function viewLabs() {
+        MyDataHelps.openApplication("https://hw.careevolutionapps.com/LabReports.html?lang=" + MyDataHelps.getCurrentLanguage());
+    }
+
+    function viewTermInfo(termInfo: TermInformation) {
+        var queryString = new URLSearchParams({ termFamily: termInfo.TermFamily, termNamespace: termInfo.TermNamespace, termCode: termInfo.TermCode, lang: MyDataHelps.getCurrentLanguage() }).toString();
+        MyDataHelps.openApplication("https://hw.careevolutionapps.com/TermInformation.html?" + queryString, { modal: true });
+    }
+
+    function viewHealthSectionDetails(concept: string) {
```

Comment:
I think the data type here can be `HealthPreviewSectionConcept` instead of `string`.